tests/std/tests/Dev11_0535636_functional_overhaul/test.cpp

```diff
@@ -1032,21 +1060,76 @@ void test_reference_wrapper_invocation() {
 
 
 // Test C++17 invoke().
+#if _HAS_CXX17
+constexpr bool test_invoke_constexpr() {
+    // MSVC++ implements LWG-2894 as a DR back to C++17

```

Comment:
We usually refer to ourselves as `MSVC` without `++`.

Reply:
You don't work for "MSVD"?